node/src/accountant/db_big_integer/big_int_db_processor.rs

```diff
@@ -71,12 +88,15 @@ impl<T: TableNameDAO> BigIntDbProcessor<T> {
     }
 }
 
-pub trait UpdateOverflowHandler<T>: Debug + Send {
-    fn update_with_overflow<'a>(
+pub trait UpdateOverflowHandler<T>: Debug + Send
+where
```

Comment:
I think, we can keep a pure function instead of a trait. It'll make it easier to maintain.

Reply:
This is an older piece of code, also written for me to allow strict testing